### DISCUSSION TITLE
Power Supply (VDDX) and GNDX of BM13xx

### DISCUSSION THREAD

**Discussion by AhmadPanogari** [rapsacw/AsiNine-Pyramid#3] [Q&A]
Hello, introduce me to an electronics hobbyist who is currently learning various things and is interested in ASIC Miner.

I've seen the schematic you made, there's something I want to ask.

1. Why is pin 33 on the first chip (which uses an oscillator) connected to GND and the other is connected to pin 34 of the other chip?
2. Why is the VSS pin on the XC6206P connected to the VCC of each chip?

Thank you.

**Comment by rapsacw** [comment]
The asics are connected in series to the power supply. For instance: the Antminer S9 has 63 asics on 1 hash board and they are connected in groups of 3 parallel, and 21 of these groups in series. Each asic gets 9v/21 = 0.42V. The VDDIO 1.8V & 0.8V are also relative to the specific asic's ground level, not to power supply ground.
The reason for connecting them in series is that is limits the demand on the power supply and pcb.

**Reply by AhmadPanogari** [reply]
I have started to understand for power series.

If VCC_3 is the series power line, why is it also connected to the VSS (XC6206P) and GND of the capacitor.

Sorry because I'm new with this.

![image](https://user-images.githubusercontent.com/17040103/209493589-66a5394f-0666-4779-bdfb-ffa6980fc5b0.png)

**Reply by rapsacw** [reply]
VCC3 is the local ground for this asic. VDDIO is +1.8V relative to this ground level. This ground level is connected to the next asic's VDD, which has a local ground of about 0.35V lower, and has a VDDIO 1.8V relative to that ground level etc..
